app/services/idv/steps/verify_base_step.rb

```diff
@@ -201,7 +201,7 @@ def enqueue_job
 
         double_address_verification =
           current_user.establishing_in_person_enrollment&.capture_secondary_id_enabled &&
-          flow_session[:pii_from_user][:same_address_as_id] == 'false'
+          flow_session[:pii_from_user][:same_address_as_id].to_s == 'false'
```

Comment:
Sorry, I'm a little confused, why are we turning `flow_session[:pii_from_user][:same_address_as_id]` to a string when we could turn the `'false'` in a boolean?

Oh wait, is it because `same_address_as_id` it's sometimes a string and sometimes a boolean?

Reply:
If it was a string and  we change it to boolean why not just use it as boolean? Or as Jack said, there is some part of the system out of our control or some transient and persisted state?

Reply:
I was thinking about that, when we load the pii from persistent store, we check and force conversion? and when we save it we do similar. Slowly all persisted state will be converted over.

Reply:
Yelp, pii is a simple hash/struct, not encapsulated object, if so, maybe easier to handle most of scenarios.